en/admin/config.rst

```diff
@@ -1650,6 +1654,14 @@ The following are parameters related to SQL statements and data types supported
       
         1, NULL
 
+**ignore_trailing_space**
+
+If this parameter's value is **yes**, the string comparison follows "trailing space insensitive" rules as below example.
+The string value 'abc' and 'abc ' is equal regardless the types of values are VARCHAR-type with variable length.
+
+**CAUTION:**
+For the string data stored as **ignore_trailing_space** is set to **no** or **yes**; will result in confusion at several parts of using the data when the configuration parameter value is changed to set to **yes** or **no**. Therefore, this configuration value change should be considered seriously. CUBRID recommends that the value of this parameter is fixed and not to change for all life times in the database.
```

Comment:
I suggest the following changes.
1. "do not change the parameter value" appears first. It is the main point of the caution.
2. "this parameter is fixed" seems to have ambiguous. it needs to give a specific moment.
3. need to give specific situations of what confusion or misbehavior is caused.